components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/dao/OAuthAppDO.java

```diff
@@ -258,4 +259,12 @@ public String getTokenType() {
     public void setTokenType(String tokenType) {
         this.tokenType = tokenType;
     }
+
```

Comment:
Add method level descriptions.

Reply:
Fixed in https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1058